### PR TITLE
isProc: recognize builtin-commands

### PR DESCRIPTION
**Pull request by giautm**
Fixed: https://github.com/microsoft/go-mssqldb/issues/251

### CODE REVIEW

**Comment by giautm** [comment]
@microsoft-github-policy-service agree

File: mssql_test.go [hunk 1]
```diff
@@ -48,6 +48,9 @@ func TestIsProc(t *testing.T) {
 		{"--proc", false},
 		{"[proc;]", true},
 		{" proc", false},
+		{"RECONFIGURE", false},
+		{"SHUTDOWN;", false},
```
Comment (shueybubbles) [inline mssql_test.go]: >"SHUTDOWN; [](http://example.com/codeflow?start=3&length=10)

perhaps add test cases for whitespace; it's not clear from the implementation of isProc that it trims.

Comment (giautm) [inline mssql_test.go]:
```suggestion
		{"SHUTDOWN", false},
```

Comment (giautm) [reply mssql_test.go]: Fixed, thank you.

Comment (giautm) [reply mssql_test.go]: @shueybubbles there is already a test for ` proc` (including space). If I passing `SHUTDOWN ` or `SHUTDOWN;` the logic of `isProc` still correct, because the space and semicolon there. This PR is for the case `SHUTDOWN` only. (no extra characters).

Comment (giautm) [reply mssql_test.go]: @shueybubbles hello, Can I have a review there, please?

**Review by shueybubbles — APPROVED** [review]
:shipit:

**Review by apoorvdeshmukh — APPROVED** [review]
:shipit:

**Comment by giautm** [comment]
@apoorvdeshmukh ping again, 😄

**Comment by shueybubbles** [comment]
@giautm do you think we should also add COMMIT here? It might be causing this bug: https://github.com/microsoft/go-sqlcmd/issues/569
